### ISSUE TITLE
Assign @some_value not found in Drab EEx template

### ISSUE THREAD

**Issue by GBH** [grych/drab#40]
It's a bit odd one for me. Let's assume we have this:

```
render(conn, "show.html", some_value: "my value")
```

If I don't somehow use that assign inside the template it will not be available for commander's `Drab.Live.peek(socket, :some_value)`

Work-around is to include it like this (even it does absolutely nothing in the template): 

```
<% @some_value # don't remove or drab will complain %>
```

**Comment by grych** [comment]
Yes, only assigns you actually use in the template are available. This is by design - eex engine only knows those assigns. There is no way to indicate the assigns you used in the runtime.

Anyway, it should be well documented, so I will leave the issue opened.

**Comment by grych** [comment]
Fixed in v0.5.4 (updated docs)
